ci-operator/step-registry/assisted/baremetal/setup/assisted-baremetal-setup-commands.sh

```diff
@@ -17,6 +17,7 @@ tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted.tar.gz"
 
 # Prepare configuration and run
 scp "${SSHOPTS[@]}" "${CLUSTER_PROFILE_DIR}/pull-secret" "root@${IP}:pull-secret"
+ssh "${SSHOPTS[@]}" "root@${IP}" "mkdir -p /root/.docker && cp /root/pull-secret /root/.docker/config.json"
```

Comment:
I keep the change quite conservative because I don't know the impact of changing the name of `pull-secret` witch is overridden below by `prod-pull-secret` and I don't know if the use of the `ENVIRONMENT` variable as well, it seems `production` is the only valid value.

Reply:
I think it cannot hurt, but my worries are that podman uses a different config location at ${XDG_RUNTIME_DIR}/containers/auth.json

Reply:
it supports `.docker/config.json` as well https://docs.podman.io/en/latest/markdown/podman-login.1.html